commit d0c41a08fba482cd0fe79acc93e1dacd7be74025 (refs/changes/07/207/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-04T22:34:59+00:00 (3 years, 2 months ago)
    
    TOOLS-1508 prep_dataset_in_jpc.sh using wrong "mantapath" dir for built image export
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index afc1332..6d2c5d9 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -362,7 +362,7 @@ if [[ -n ${SDC_LOCAL_BUILD} ]]; then
     curl -sS -f -o ${output_dir}/${image_filename} ${SDC_IMGAPI_URL}/images/${image_id}/file
 
 else
-    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | cut -d '-' -f1,2)/${build_name}
+    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | sed -E 's/-g[a-f0-9]{7,40}$//')/${build_name}
     mmkdir -p ${mantapath}
 
     manta_bits=/tmp/manta-exported-image.$$
